src/user-points/user-points.jobs.controller.ts

```diff
@@ -68,4 +71,55 @@ export class UserPointsJobsController {
     await this.userPointsService.upsert(options);
     return { requeue: false };
   }
+
+  @MessagePattern(GraphileWorkerPattern.REFRESH_POOL_4_POINTS)
+  @UseFilters(new GraphileWorkerException())
+  async refreshPool4Points({
+    userId,
+    endDate,
+  }: RefreshPool4Options): Promise<GraphileWorkerHandlerResponse> {
+    const user = await this.usersService.find(userId);
+    const end = endDate ?? new Date();
```

Comment:
This is purely here for testing, standard codepath will always land with `new Date()`